argocd/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "49ad19d0-1452-4275-b0fe-cbda3821807d",
+  "app_id": "argocd",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Argo CD",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "Argo CD",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "argocd.",
+        "check": [
+          "argocd.api_server.go.goroutines",
+          "argocd.app_controller.go.goroutines",
+          "argocd.repo_server.go.goroutines"
+        ],
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {},
+    "monitors": {},
+    "logs": {},
```

Comment:
I think you can remove this since you don't have a logs pipeline yet and it should fix the manifest validation error.